classes/submission/Repository.php

```diff
@@ -26,9 +26,10 @@
 use Illuminate\Support\Enumerable;
 use Illuminate\Support\LazyCollection;
 use PKP\context\Context;
+use PKP\context\PKPSection;
```

Comment:
This class should not exist any more, after the conversion to the new Repository/DAO/Map pattern.
I see also some old paths in OJS... They will probably go away once you rebase again, but be careful if there are any conflicts... :-)

Reply:
Not sure why that was added, but I've rebased and this use statement was the only reference to it. I've removed it.